test/unit/services/JobRoleServiceTests.ts

```diff
@@ -258,5 +263,49 @@ describe('JobRoleService', function () {
       })
 
     })
+    describe('Generate A Report of All Job Roles', function () {
+
+      it('should throw an error when the API returns a 200 status', async () => {
```

Comment:
Why does this 200 status test throw an error?

Reply:
Fixed